### ISSUE TITLE
Do `reflect` and `axiomatize` do the same thing?

### ISSUE THREAD

**Issue by awstlaur** [ucsd-progsys/liquidhaskell#882]
Looking at Parse.hs, they appear to be synonyms. Why have them both?

**Comment by nikivazou** [comment]
They do exaclty the same thing. 

We started using `axiomatize` and wrote a lot of code with it. Then we switced to `reflect` but kept `axiomatize` for compatibility.

**Comment by ranjitjhala** [comment]
Good point; purely for historical reasons. We should replace axiomatize
with reflect and deprecate the former.
